src/main/java/frc/robot/Constants.java

```diff
@@ -29,11 +31,15 @@ public static class Ports {
     public static final class Swerve {
         public static final boolean GYRO_INVERSION = false; // Always ensure Gyro is CCW+ CW-
 
+        public static final PIDConstants AUTO_TRANSLATION = new PIDConstants(0);
```

Comment:
nit: I like to leave `// TODO` comments on stubbed methods and improperly set comments so it's easy to grep what you haven't filled out yet later